lnd_test.go

```diff
@@ -3801,69 +3778,110 @@ func testSpiderShortestPath(net *lntest.NetworkHarness, t *harnessTest) {
 				nodeNames[connections[i][1]], nodeNames[connections[i][0]], err)
 		}
 	}
-	
 
-	// Open up a payment streams to each node, that we'll use to
-	// send payment between nodes.
 
-	payStreams := make([]lnrpc.Lightning_SendPaymentClient, numNodes)
+	timeLockDelta := uint32(144)
+
+	expectedPolicy := &lnrpc.RoutingPolicy{
+		FeeBaseMsat:      baseFee,
+		FeeRateMilliMsat: feeRate * testFeeBase,
+		TimeLockDelta:    timeLockDelta,
+		MinHtlc:          1000, // default value
+	}
 
+	graphSubs := make([]graphSubscription, numNodes)
```

Comment:
Can you add some more comments on what exactly is happening here? Did you add these graphNotifications? If so, what are they/where are they? If not, can you expand on how you're using them?